game_eggs/steamcmd_servers/palworld/README.md

```diff
@@ -98,4 +98,21 @@ As of January 19th 2024 the server requires about 5GB of storage capacity. This
 | Game            | 8211    |
 | RCON (optional) | 25575   |
 
-The RCON port does not need to be allocated.
\ No newline at end of file
+The RCON port does not need to be allocated.
+
+### Updating
+
+1. Update your egg
+2. Update the startup of all already made servers to the one that now comes with the egg
+3. Then hit reinstall because the parser application has to be downloaded
```

Comment:
Also worth calling out if saves will be wiped here, and if how to restore them - or if reinstall is simply for config/images/binaries and not the mounted directory for the saves